data/json/items/comestibles/baked.json

```diff
@@ -18,5 +18,24 @@
     "stack_size": 24,
     "fun": 3,
     "vitamins": [ [ "vitC", 3 ], [ "calcium", 2 ] ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "hazelnut_cake",
+    "name": { "str_sp": "hazelnut cake" },
+    "weight": "30 g",
+    "color": "brown",
+    "spoils_in": "7 days",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "calories": 160,
+    "description": "This traditional hazelnut cake (or Bag8nizab8nak cake in Abenaki) is a nice, protein-rich treat for cold days.",
```

Comment:
![image](https://user-images.githubusercontent.com/99621099/198842327-839dbbb0-31cc-4a1d-b721-d663469bcf90.png)

I don't think those accented letters worked 🤔

Reply:
Yeah, that actually was kind of intentional. Apparently, there is an extra letter in Algonquian languages that looks almost like an 8 and a recipe source I used actually just put an 8 where the letter should be. This one ȣ. So I did the same, assuming at first the letter is actually just an 8... Not sure if cataclysm can depict special characters like that though?! I'll try it out.

Reply:
seems to work now with the special characters!